backend/src/db-utils.js

```diff
@@ -322,11 +346,13 @@ const aggregateStats = async (options) => {
     queryLatestBlockHeight(options),
   ]);
   return {
-    totalAccounts: totalAccounts.total,
-    totalBlocks: totalBlocks.total,
-    totalTransactions: totalTransactions.total,
-    lastDayTxCount: lastDayTxCount.total,
-    lastBlockHeight: lastBlockHeight ? lastBlockHeight.block_height : 0,
+    totalAccounts: parseInt(totalAccounts.total),
+    totalBlocks: parseInt(totalBlocks.total),
+    totalTransactions: parseInt(totalTransactions.total),
+    lastDayTxCount: parseInt(lastDayTxCount.total),
+    lastBlockHeight: lastBlockHeight
+      ? parseInt(lastBlockHeight.block_height)
+      : 0,
```

Comment:
Postgres returns these as strings (I assume it uses int64 numbers here, which don't fit into JS number in the worst-case scenario, but we don't expect any of the values to overflow over 2^53 any time soon)